README.md

```diff
@@ -1,10 +1,14 @@
 ![Spyder — The Scientific Python Development Environment](
 ./img_src/spyder_readme_banner.png)
 
-*Copyright © 2009–2018 Spyder Project Contributors*
+*Copyright © 2009–2018 [Spyder Project Contributors](
```

Comment:
Do you we need to leave an ending year? You know I really don't agree with it because updating the ending year is just more work for so little gain.

Reply:
I eliminated an ending year everywhere else, but the readme is the one single place it makes sense to keep it, IMO (it was already there after all). Its the thing that many people are likely to look at and for many is the first impression people get of the project, so including the current year front and center makes clear that our project is active and up to date, a positive effect (if one considers the tens or even hundreds of thousands of times people will see it) that far outweighs a few seconds once per year, particularly relative to the alternative, only listing the begin year and a dash which at a quick first glance could look like just "2009".

Reply:
People can take a look at the Contributors graph to see if the project is still active. I see no gain for this, other than more work for me.

So I vote for removing the ending year.

Reply:
> People can take a look at the Contributors graph to see if the project is still active. 

This serves a totally different purpose, and in no way substitutes for it being in copyright line at the top of the readme. The whole point is so someone seeing the readme and perhaps the project for the first time to (whether or not on Github) can *at first glance* tell that it is active and up to date. Requiring them to be on Github, know how to access it, *want* to access it, put in the effort to do so, wait ~5 or more seconds for it to load, and then interpret it (when it is to *our* benefit as well as theirs for it to be immediately apparent to all) is not very sensible, IMO.

> I see no gain for this, other than more work for me.

It only need (and should) be updated once another change is made anyway in a new year, so any overhead on top of the trivial time to change a single digit at the top of a repo-root-directory file is minimal. I myself am more than willing to be responsible for it, or even fund Spyder an extra $1 per year which should more than pay for the time ~60 or so seconds per year it should take. Relative to that, it seems highly improbable that the small but positive boost in the impression of our project multiplied across >=10,000s of people seeing it wouldn't be worth it in comparison.

Reply:
> This serves a totally different purpose, and in no way substitutes for it being in copyright line at the top of the readme. The whole point is so someone seeing the readme and perhaps the project for the first time to (whether or not on Github) can at first glance tell that it is active and up to date. Requiring them to be on Github, know how to access the Contrib Graph, want to access it, put in the effort to do so, wait ~5 or more seconds for it to render, and then interpret it (when it is to our benefit as well as theirs for it to be immediately apparent to all) is not very sensible, IMO.

I always look at the last commit date to see if a project is still active or not. No need to be logged in and it appears at the top of the project page on GitHub. The copyright info doesn't mean as much to me as a real tangible action that happened on the project.

![image](https://user-images.githubusercontent.com/10170372/45265445-dde83e80-b418-11e8-8f5c-d7a9466be1c0.png)

Reply:
> I always look at the last commit date to see if a project is still active or not. 

Yeah, I also do the same thing, with the copyright date if provided offering secondary confirmation. Many people, particularly those newer to Github and scientific development (i.e. coming across Spyder for the first time, looking for a data science IDE, might not know to look there (its rather small text in the corner), or even know how to interpret it, so it still provides some value, particularly to the people we want to make a good first impression on. It also is present on all places where someone would find the readme (in the package itself, downstream distributors, etc) whereas the last commit is only on Github.

> However, I understand that it is an additional task to keep track on for @ccordoba12, and small tasks do add up in the end, so I respect the fact that the final word on this is yours.

Agreed. @ccordoba12 final call?

Reply:
Let's leave it. Since it's the only place that needs to be updated, it's not that hard. And hopefully someone else remembers to do it.